src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyPage.tsx

```diff
@@ -110,7 +110,7 @@ const MethodologyPage: NextPage<Props> = ({ data }) => {
                   </div>
 
                   <div className="govuk-grid-column-three-quarters">
-                    <SectionBlocks content={content} />
+                    <SectionBlocks releaseId="" content={content} />
```

Comment:
Shouldn't be passing through empty strings. Make `releaseId` optional in `SectionBlocks`.